pkg/diffs/diffs.go

```diff
@@ -166,6 +191,15 @@ func GetPresubmitsForCiopConfigs(prowConfig *prowconfig.Config, ciopConfigs conf
 				}
 				if config.IsCiopConfigCM(env.ValueFrom.ConfigMapKeyRef.Name) {
 					if _, ok := ciopConfigs[env.ValueFrom.ConfigMapKeyRef.Key]; ok {
+						testName := strings.TrimPrefix(job.Name, "pull-ci-")
+						orgRepo := strings.Replace(repo, "/", "-", -1)
+						testName = strings.TrimPrefix(testName, fmt.Sprintf("%s-%s-", orgRepo, job.Brancher.Branches[0]))
```

Comment:
I am a little worried because `job.Brancher.Branches[0]` is a regex, but we generate presubmits as plain names anyway...